api4/system.go

```diff
@@ -395,6 +396,10 @@ func testS3(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if cfg.FileSettings.AmazonS3SecretAccessKey != "" && strings.ContainsAny(cfg.FileSettings.AmazonS3SecretAccessKey, "*") {
```

Comment:
This can be simplified to just

```go
if strings.ContainsAny(cfg.FileSettings.AmazonS3SecretAccessKey, "*")
```

Reply:
thanks @ccbrown. using the suggestion made by @saturninoabril